ansible/roles/awx_pod/defaults/main.yml

```diff
@@ -0,0 +1,64 @@
+---
+
+#
+# container image settings
+#
+
+#awx_awxweb_version: 10.0.0
+#awx_awxtask_version: 10.0.0
+awx_awxweb_version: 11.2.0
+awx_awxtask_version: 11.2.0
+awx_postgres_version: 10
+
+awx_awxweb_image: docker.io/ansible/awx_web:{{ awx_awxweb_version }}
+awx_awxtask_image: docker.io/ansible/awx_task:{{ awx_awxtask_version }}
+awx_postgres_image: docker.io/centos/postgresql-{{ awx_postgres_version }}-centos7
+awx_memcached_image: docker.io/library/memcached:alpine
+awx_redis_image: docker.io/library/redis:latest
+
+# Update postgresql from a lower version
+awx_postgres_update: False
+
+#
+# awx host settings
+#
+
+admin_user: admin
+admin_email: "{{ awx_admin_email | default('root@localhost') }}"
+admin_password: "{{ awx_admin_password |default('password') }}"
+
+awx_pod_name: awx
+awx_pod_label: awx
+
+awx_podman_dir: /tmp
+
+awx_host_port: 80
+awx_host_ssl_port: 443
+
+# Optional SSL stuff
+# ssl_certificate: /etc/cockpit/ws-certs.d/0-self-signed.cert
+# ca_trust_file: /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem
+
+#
+# anything below is pod internal info, never seen outside.
+#
+pg_hostname: localhost
+pg_port: 5432
+pg_database: awx
+pg_username: awx
+pg_password: awxpass
```

Comment:
can password default values please be automatically generated instead?

Reply:
changed to use `"{{ awx_admin_password | default('password') }}"` instead